docs/blog/oras-0.15-a-fully-functional-registry-client.md

```diff
@@ -0,0 +1,189 @@
+
+```
+$ oras manifest fetch localhost:5000/oras-repo/hello-world:latest
+{
+   "schemaVersion": 2,
+   "mediaType": "application/vnd.oci.image.manifest.v1+json",
+   "config": {
+      "mediaType": "application/vnd.oci.image.config.v1+json",
+      "size": 1469,
+      "digest": "sha256:feb5d9fea6a5e9606aa995e879d862b825965ba48de054caab5ef356dc6b3412"
+   },
+   "layers": [
+      {
+         "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+         "size": 2479,
+         "digest": "sha256:2db29710123e3e53a794f2694094b9b4338aa9ee5c40b930cb8063a1be392c54"
+      }
+   ]
+}
+```
+
+Run and validate the new OCI image:
+
+```
+$ docker run localhost:5000/oras-repo/hello-world:latest
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (amd64)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
+
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
+
+Share images, automate workflows, and more with a free Docker ID:
+ https://hub.docker.com/
+
+For more examples and ideas, visit:
+ https://docs.docker.com/get-started/
+
+```
+
+It turns out the conversion works.
+
+## Try multi-tagging and view the tags 
+
+Tag the manifest with 'latest' to 'v1.0.0', 'v2.0.0':
+
+```
+$ oras tag localhost:5000/oras-repo/hello-world:latest v1.0.0 v2.0.0
+```
+
+View the newly created tags in the sample repo:
+
+```
+$ oras repo show-tags localhost:5000/oras-repo/hello-world
+latest
+v1.0.0
+v2.0.0
+```
+
+Congratulations! You have experienced all new top-level commands in ORAS CLI 0.15. 
+
+## What's next for ORAS
+
+Recently, the ORAS maintainers submitted the proposal via [pr-934](https://github.com/opencontainers/image-spec/pull/934) and [pr-335](https://github.com/opencontainers/distribution-spec/pull/335) to OCI which provides a solution that adds new capabilities to OCI registries while maintaining the ability to function on existing registries. This proposal originates from the [ORAS artifact spec](https://github.com/oras-project/artifacts-spec) and has been accepted by the OCI group. OCI group also cut new release in distribution-spec and image-spec to support Reference Types. 
+
+In ORAS CLI 0.16 and ORAS-go v2.0.0-rc.4, support OCI artifact spec will be the most significant plan and is targeted to be released at the end of October. You can find the migration proposal from this [doc](https://hackmd.io/zLnCh8WRQDG-3S_jXR626g?view). See also the [ORAS Roadmap](https://github.com/oras-project/community/blob/main/Roadmap.md) for more detailed future plans.
```

Comment:
Suggest adding links to the milestones and tweak the wording a bit:
In [ORAS CLI 0.16](https://github.com/oras-project/oras/milestone/9) and [ORAS-go v2.0.0-rc.4](https://github.com/oras-project/oras-go/milestone/10), we'll add support for the [OCI artifact spec](https://github.com/opencontainers/image-spec/blob/main/artifact.md) will be the most...